src/tables/cblc.rs

```diff
@@ -45,24 +59,28 @@ fn select_bitmap_size_table(
 
     let mut idx = None;
     let mut max_ppem = 0;
+    let mut max_bit_depth = 0;
     for i in 0..subtable_count {
         // Check that the current subtable contains a provided glyph id.
         s.advance(40); // Jump to `start_glyph_index`.
         let start_glyph_id = s.read::<GlyphId>()?;
         let end_glyph_id = s.read::<GlyphId>()?;
-        let ppem = u16::from(s.read::<u8>()?);
+        let ppem_x = u16::from(s.read::<u8>()?);
+        s.advance(1); // ppem_y
+        let bit_depth = s.read::<u8>()?;
+        s.advance(1); // flags
 
         if !(start_glyph_id..=end_glyph_id).contains(&glyph_id) {
-            s.advance(4); // Jump to the end of the subtable.
             continue;
         }
 
         // Select a best matching subtable based on `pixels_per_em`.
-        if (pixels_per_em <= ppem && ppem < max_ppem)
-            || (pixels_per_em > max_ppem && ppem > max_ppem)
+        if (pixels_per_em <= ppem_x && ppem_x < max_ppem)
+            || (pixels_per_em > max_ppem && ppem_x > max_ppem)
         {
             idx = Some(usize::num_from(i));
-            max_ppem = ppem;
+            max_ppem = ppem_x;
+            max_bit_depth = bit_depth;
```

Comment:
Is the last subtable expected to have the biggest bit depth?

Reply:
Nope, renamed to `bit_depth_for_max_ppem` to clarify.